server/scheduled_executor_service.go

```diff
@@ -27,13 +27,15 @@ func NewScheduledExecutorService(
 	workingDirIterator events.WorkDirIterator,
 	statsScope stats.Scope,
 	log logging.SimpleLogging,
-	pullCleaner events.PullCleaner,
+	closedpullCleaner events.PullCleaner,
+	openpullCleaner events.PullCleaner,
```

Comment:
Looks like this is openPullCleaner/clostedPullCleaner with capital P everywhere else. Not sure if that is just to avoid naming collision but its a little confusing.

Reply:
Will fix.